### PR TITLE
chore: remove empty files

### PR DESCRIPTION
**Pull request by thinkingserious**
### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license

Internal Reference: DI-1044

Associated PRs:
- https://github.com/twilio/terraform-provider-twilio/pull/32/files

### CODE REVIEW

File: src/main/resources/terraform-provider-twilio/api.mustache [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 )
 
 {{#resources}}
+{{^readOnly}}
```
[the remark this answers is not in the document]
Comment (thinkingserious) [reply src/main/resources/terraform-provider-twilio/api.mustache:14]: To confirm, we don't need to support resources that only have partial CRUD operations defined?

[the remark this answers is not in the document]
Comment (shwetha-manvinkurke) [reply src/main/resources/terraform-provider-twilio/api.mustache:14]: That's my understanding and besides terraform doesn't allow partial CRUD operations.

File: src/main/java/com/twilio/oai/TwilioTerraformGenerator.java [hunk 2]
```diff
@@ -119,7 +119,16 @@ private void populateCrudOperations(Map<String, Object> resource, String operati
...
+        resource.put("hasAllCrudOps",
+            (Boolean) resource.getOrDefault("hasCreate", false) &&
+            (Boolean) resource.getOrDefault("hasRead", false) &&
+            (Boolean) resource.getOrDefault("hasUpdate", false) &&
+            (Boolean) resource.getOrDefault("hasDelete", false));
+
+        resource.put("readOnly",
+                (!((Boolean) resource.getOrDefault("hasCreate", false)) &&
+                 !((Boolean) resource.getOrDefault("hasUpdate", false)) &&
+                 !((Boolean) resource.getOrDefault("hasDelete", false))));
```
Comment (childish-sambino) [inline src/main/java/com/twilio/oai/TwilioTerraformGenerator.java:131]: I don't see this logic as necessary. Can't we just check for `hasAllCrudOps`?

File: src/main/resources/terraform-provider-twilio/api.mustache (hunk 1 above)
Comment (childish-sambino) [inline src/main/resources/terraform-provider-twilio/api.mustache:14]: I'd prefer logic in the generator that removes these resources. Then we don't need to delete the files later.

File: examples/build_twilio_go.py [hunk 3]
```diff
@@ -34,6 +35,13 @@ def generate(openapi_spec_path, go_path, domain, is_file=False, language='go'):
               f"{go_path}/{sub_dir}/{domain_name}/{api_version}"
     os.system(command)
 
+    # Find files that only have read-only operations
+    files = glob.glob(f"{go_path}/{sub_dir}/**/*.*", recursive=True)
```
Comment (JenniferMah) [inline examples/build_twilio_go.py:39]: Will this also remove all read only operations for twilio-go or just for terraform?

**Review by JenniferMah — COMMENTED** [review]
Is there a PR for the corresponding changes after generating twilio go?

File: examples/build_twilio_go.py (hunk 3 above)
Comment (thinkingserious) [reply examples/build_twilio_go.py]: Yes! Great catch, fixed.

**Comment by thinkingserious** [comment]
Added [PR](https://github.com/twilio/terraform-provider-twilio/pull/26) [looks like there may be an unrelated error] for corresponding changes. There were no changes to `twilio-go`.

File: src/main/resources/terraform-provider-twilio/api.mustache (hunk 1 above)
Comment (childish-sambino) [reply src/main/resources/terraform-provider-twilio/api.mustache]: Is this still WIP?

Comment (thinkingserious) [reply src/main/resources/terraform-provider-twilio/api.mustache]: Yes, but not on this file. I no longer think the logic should be in the template.

Comment (thinkingserious) [reply src/main/resources/terraform-provider-twilio/api.mustache]: That is also my preference. I'm exploring using the `.openapi-generator-ignore` [feature](https://openapi-generator.tech/docs/faq-extending/#how-do-i-skip-files-during-code-generation).

Comment (thinkingserious) [reply src/main/resources/terraform-provider-twilio/api.mustache]: These are the options so far:
1. post-process the files as demonstrated in this PR
2. pre-process the files and write a list of files to ignore to `.openapi-generator-ignore`
3. dig deeper on how the `openapi-generator` determines what resources to process and filter there (already spent a good amount of time researching this with no solution)

Comment (childish-sambino) [reply src/main/resources/terraform-provider-twilio/api.mustache]: You can process the OpenAPI doc beforehand and remove any operations that correspond to a resource without all the CRUD operations. Example of how to process the doc: https://github.com/twilio/twilio-oai-generator/blob/1fdd14860564bb962a3c9870d10916df1a34c36d/src/main/java/com/twilio/oai/TwilioNodeGenerator.java#L56

File: src/main/java/com/twilio/oai/TwilioTerraformGenerator.java [hunk 4]
```diff
@@ -23,6 +25,21 @@ public TwilioTerraformGenerator() {
         typeMapping.put("object", "string");
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public void processOpenAPI(OpenAPI openAPI) {
+        super.processOpenAPI(openAPI);
+        AtomicBoolean allCrudAvailable = new AtomicBoolean(false);
+        openAPI.getPaths().forEach((name, path) -> path.readOperations().forEach(operation -> {
```
Comment (childish-sambino) [inline src/main/java/com/twilio/oai/TwilioTerraformGenerator.java:33]: No need to iterate over `readOperations()` as it's not used. Recommend installing the SonarQube plugin for IntelliJ. It warns about such things.

**Review by childish-sambino — APPROVED** [review]

**Review by JenniferMah — APPROVED** [review]

File: src/main/java/com/twilio/oai/TwilioTerraformGenerator.java [hunk 5]
```diff
@@ -62,6 +64,21 @@ private boolean containsResponseProperty(final Operation operation, final String
                 .anyMatch(properties -> properties.contains(StringUtils.underscore(propertyName))));
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public void processOpenAPI(OpenAPI openAPI) {
```
Comment (childish-sambino) [inline src/main/java/com/twilio/oai/TwilioTerraformGenerator.java:69]: Need to merge this with `processOpenAPI` above.

**Pull request merged**